terasoluna-tourreservation-selenium/src/test/resources/META-INF/spring/seleniumContext.xml

```diff
@@ -20,5 +20,6 @@
 
   <bean id="dateFactory" class="org.terasoluna.gfw.common.date.jodatime.DefaultJodaTimeDateFactory" />
 
+  <bean id="webDriverListenerImpl" class="org.terasoluna.tourreservation.tourreserve.selenium.WaitWebDriverEventListener" />
```

Comment:
- [ ] Bean id is not fixed yet.

Reply:
fixed bean name which modification was missing.
b1f2718